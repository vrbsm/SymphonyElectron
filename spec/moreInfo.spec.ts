import { shallow } from 'enzyme';
import * as React from 'react';
import MoreInfo from '../src/renderer/components/more-info';

describe('more info', () => {
    it('should render correctly', () => {
        const wrapper = shallow(React.createElement(MoreInfo));
        expect(wrapper).toMatchSnapshot();
    });
});
