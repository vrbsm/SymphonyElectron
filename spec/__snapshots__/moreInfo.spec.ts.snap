// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`more info should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MoreInfo />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <span>
          <b>
            Version Information
          </b>
        </span>,
        <span
          className="MoreInfo-electron"
        />,
        <span
          className="MoreInfo-chrome"
        />,
        <span
          className="MoreInfo-v8"
        >
          6.8.275.32-node.45
        </span>,
        <span
          className="MoreInfo-node"
        >
          10.15.0
        </span>,
        <span
          className="MoreInfo-openssl"
        >
          1.1.0j
        </span>,
        <span
          className="MoreInfo-zlib"
        >
          1.2.11
        </span>,
        <span
          className="MoreInfo-uv"
        >
          1.23.2
        </span>,
        <span
          className="MoreInfo-ares"
        >
          1.15.0
        </span>,
        <span
          className="MoreInfo-http_parser"
        >
          2.8.0
        </span>,
      ],
      "className": "MoreInfo",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <b>
            Version Information
          </b>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Version Information",
          },
          "ref": null,
          "rendered": "Version Information",
          "type": "b",
        },
        "type": "span",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "MoreInfo-electron",
        },
        "ref": null,
        "rendered": null,
        "type": "span",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "MoreInfo-chrome",
        },
        "ref": null,
        "rendered": null,
        "type": "span",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "6.8.275.32-node.45",
          "className": "MoreInfo-v8",
        },
        "ref": null,
        "rendered": "6.8.275.32-node.45",
        "type": "span",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "10.15.0",
          "className": "MoreInfo-node",
        },
        "ref": null,
        "rendered": "10.15.0",
        "type": "span",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "1.1.0j",
          "className": "MoreInfo-openssl",
        },
        "ref": null,
        "rendered": "1.1.0j",
        "type": "span",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "1.2.11",
          "className": "MoreInfo-zlib",
        },
        "ref": null,
        "rendered": "1.2.11",
        "type": "span",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "1.23.2",
          "className": "MoreInfo-uv",
        },
        "ref": null,
        "rendered": "1.23.2",
        "type": "span",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "1.15.0",
          "className": "MoreInfo-ares",
        },
        "ref": null,
        "rendered": "1.15.0",
        "type": "span",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "2.8.0",
          "className": "MoreInfo-http_parser",
        },
        "ref": null,
        "rendered": "2.8.0",
        "type": "span",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <span>
            <b>
              Version Information
            </b>
          </span>,
          <span
            className="MoreInfo-electron"
          />,
          <span
            className="MoreInfo-chrome"
          />,
          <span
            className="MoreInfo-v8"
          >
            6.8.275.32-node.45
          </span>,
          <span
            className="MoreInfo-node"
          >
            10.15.0
          </span>,
          <span
            className="MoreInfo-openssl"
          >
            1.1.0j
          </span>,
          <span
            className="MoreInfo-zlib"
          >
            1.2.11
          </span>,
          <span
            className="MoreInfo-uv"
          >
            1.23.2
          </span>,
          <span
            className="MoreInfo-ares"
          >
            1.15.0
          </span>,
          <span
            className="MoreInfo-http_parser"
          >
            2.8.0
          </span>,
        ],
        "className": "MoreInfo",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <b>
              Version Information
            </b>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Version Information",
            },
            "ref": null,
            "rendered": "Version Information",
            "type": "b",
          },
          "type": "span",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "MoreInfo-electron",
          },
          "ref": null,
          "rendered": null,
          "type": "span",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "MoreInfo-chrome",
          },
          "ref": null,
          "rendered": null,
          "type": "span",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "6.8.275.32-node.45",
            "className": "MoreInfo-v8",
          },
          "ref": null,
          "rendered": "6.8.275.32-node.45",
          "type": "span",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "10.15.0",
            "className": "MoreInfo-node",
          },
          "ref": null,
          "rendered": "10.15.0",
          "type": "span",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "1.1.0j",
            "className": "MoreInfo-openssl",
          },
          "ref": null,
          "rendered": "1.1.0j",
          "type": "span",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "1.2.11",
            "className": "MoreInfo-zlib",
          },
          "ref": null,
          "rendered": "1.2.11",
          "type": "span",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "1.23.2",
            "className": "MoreInfo-uv",
          },
          "ref": null,
          "rendered": "1.23.2",
          "type": "span",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "1.15.0",
            "className": "MoreInfo-ares",
          },
          "ref": null,
          "rendered": "1.15.0",
          "type": "span",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "2.8.0",
            "className": "MoreInfo-http_parser",
          },
          "ref": null,
          "rendered": "2.8.0",
          "type": "span",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
